package.json

```diff
@@ -9,18 +9,28 @@
     "type": "git",
     "url": "https://github.com/Microsoft/napajs.git"
   },
+  "binary": {
+    "module_name": "napa-binding",
```

Comment:
Does this include all files in `bin` directory? in that case I suggest to use `napa-bin`